core/src/test/java/io/methvin/watchservice/DirectoryWatcherOnDiskTest.java

```diff
@@ -30,7 +35,7 @@
   public void setUp() throws IOException {
     this.tmpDir = Files.createTempDirectory(null);
     this.recorder = new EventRecorder();
-    this.watcher = DirectoryWatcher.create(this.tmpDir, this.recorder);
+    this.watcher = DirectoryWatcher.create(Collections.singletonList(this.tmpDir), this.recorder, false);
```

Comment:
It might be worth running all these tests with both `true` and `false` to make sure both work. What do you think?